src/zip.js

```diff
@@ -36,14 +36,18 @@ module.exports = function (gj, options, stream = false) {
     }
   });
 
-  var generateOptions = { type: "base64", compression: "STORE" };
+  var generateOptions = { };
+  if (options && options.type) {
+    generateOptions.type = options.type;
```

Comment:
Changing this line throws: `Error: No output type specified.`

It seems that `options.type` needs to be required and documentation added to the README.

Reply:
Ended up adding a default and making the property required in the interface in the types file, good call!